src/components/Graph.js

```diff
@@ -110,13 +117,13 @@ export class Graph extends React.Component {
 
   nodeMouseOver(nodeGroup) {
     const multiplier = 1.5;
-    nodeGroup.select('circle').attr('r', this.nodeCircleRadius * multiplier);
-    nodeGroup.select('text').attr('font-size', `${this.nodeIconFontSize * multiplier}px`);
+    nodeGroup.selectAll('circle').attr('r', this.nodeCircleRadius * multiplier);
+    nodeGroup.selectAll('text').attr('font-size', `${this.nodeIconFontSize * multiplier}px`);
```

Comment:
It looks like that ``font-size`` of label should be changed after mouse over.

![FontSize](https://user-images.githubusercontent.com/37223468/92567563-2b088080-f27e-11ea-9365-c2e263fcd31a.png)

Reply:
Initially only the icon (which also is `text`) was enlarged on hover. I added label enlargement with newest commit. Now it should also be clear what is enlarged where as I use class names instead of generic `text` selector.

Reply:
Oh yes, you are right. Thanks for developing this feature too!